src/hocs/withCustomNetworkDialog.js

```diff
@@ -0,0 +1,11 @@
+import { connect } from 'react-redux';
+
+import { confirm } from '../actions/dialogsActions';
+
+export default function withCustomNetworkDialog(propName = 'confirm') {
+  const mapDispatchToProps = (dispatch) => ({
+    [propName]: (...args) => dispatch(confirm(...args))
+  });
+
+  return connect(null, mapDispatchToProps);
+}
```

Comment:
Is this used? I don't see any references to it in the code

Reply:
Yep, its not used, I'll removed it